src/gui_common/MainMenu.tscn

```diff
@@ -324,6 +325,17 @@ __meta__ = {
 
 [node name="GUIAnimations" type="AnimationPlayer" parent="."]
 anims/MenuSlide = SubResource( 1 )
+
+[node name="GLES2 Popup" type="ConfirmationDialog" parent="."]
```

Comment:
Please change the type here to be a box with just the ok button as it seems the cancel / ok button don't do anything different.